.circleci/config.yml

```diff
@@ -146,7 +152,7 @@ jobs:
             - .yarn/cache
             - .yarn/unplugged
 
-      - run: 'dockerize -wait tcp://localhost:5432 -timeout 1m'
+      - run: 'dockerize -wait tcp://localhost:5432 -wait tcp://localhost:6379 -timeout 1m'
```

Comment:
This was missing, waiting for redis to be ready. Luckily it wasn't causing issues so far.